app/views/public_pages/home.html.erb

```diff
@@ -12,7 +12,10 @@
       </div>
     </div>
   <% end %>
-  <% if open_for_gyr_intake? %>
+  <% if show_offseason_filing_banner? %>
```

Comment:
AL says: Super awesome refactor idea: Make a method that returns _which_ banner to show. Then that thing can be unit tested, which avoids the need for a feature spec.

Even if that doesn't happen: Gotta include this on intro page.